pagopa/io-app
# fix: [IABT-1438] Implements zendesk Custom fields for payment log

## Short description
This PR implements new zendesk custom fields to make the payment log easier to understand on assistance

## List of changes proposed in this pull request
- map new fields ID
- send payment informations on specific fields on any assistance request functions

## How to test
Open a new assistance request from a failed payment source, check the specification to be respected
